physical_property/logP/logP_challenge_instructions.md

```diff
@@ -18,19 +24,28 @@ Octanol may be found in the aqueous phase. The mole fraction of water in octanol
+- For the "Method Category" section please state if your prediction method can be better classified as an empirical modeling method, physical quantum mechanics (QM) modeling method, physical molecular mechanics (MM) modeling method, or mixed (both empirical and physical), using the category labels `Empirical`, `Physical (MM)`, `Physical (QM)`, or `Mixed`. Empirical models are prediction methods that are trained on experimental data, such as QSPR, machine learning models, artificial neural networks etc. Physical models are prediction methods that rely on the physical principles of the system, such as molecular mechanics or quantum mechanics based methods to predict molecular properties. If your method takes advantage of both kinds of approaches please report it as “Mixed”. If you choose the “Mixed” category, please explain your decision in the beginning of Method Description section.
 
 - Names of the prediction files must have three sections separated by a `-`: predicted property `logP`, and your name and must end with an integer indicating the number of prediction set. For example, if you want to submit one prediction, you would name it `logP-myname-1.csv`, where `myname` is arbitrary text of your choice. If you submit three prediction files, you would name them `logP-myname-1.csv`, `logP-myname-2.csv`, and `logP-myname-3.csv`.
 
 - Prediction files will be machine parsed, so correct formatting is essential. Files with the wrong format will not be accepted.
 
+## Multiple submissions
+As per our policy on multiple submissions, each participant or organization is allowed only one ranked submission, which must be clearly indicated as such by filling the appropriate field in the submission form. We also accept non-ranked submissions, which we will not formally judge. These allow us to certify that your calculations were done without knowing the answers, but do not receive formal ranking, as discussed at the link above.
+
+If multiple submissions are incorrectly provided as "ranked" by a single participant, we will judge only one of them; likely this will be the first submitted, but it may be a random submission.
+
+## Predictions using multiple microstates
+If you have evaluated additional microstates for some of the molecules, we ask that you use the following formating for these predictions:
+`ID tag`, `Molecule ID` `Molecule ID of additional microstate`, `TFE`, `TFE SEM`, `TFE model uncertainty`.
+Note, there is no comma between `Molecule ID` and `Molecule ID of additional microstate`, this is for submission handling purposes. If you have evaluated additional microstates then the molecule ID of the additional microstate needs to be in the format: `SMXX_extra001` (number can vary). If multiple microstates are used, please report the order of population in the aqueous phase in descending order. Please list your chosen microstate populations and SMILES strings in the `METHOD DESCRIPTION SECTION` in your submission file.
```

Comment:
See comment on example format, concerning the naming of the second set of fields.

Reply:
Updated